llvm/lib/TableGen/TGLexer.cpp

```diff
@@ -664,11 +663,10 @@ bool TGLexer::prepExitInclude(bool IncludeStackMustBeEmpty) {
   PrepIncludeStack.pop_back();
 
   if (IncludeStackMustBeEmpty) {
```

Comment:
`IncludeStackMustBeEmpty` is only used in this if. Maybe just remove the parameter and the if/assert?

Reply:
Yeah, I wasn't sure how far to go with cleanups like that in this patch.